python/pyspark/mllib/recommendation.py

```diff
@@ -234,11 +238,35 @@ def _prepare(cls, ratings):
     def train(cls, ratings, rank, iterations=5, lambda_=0.01, blocks=-1, nonnegative=False,
               seed=None):
         """
-        Train a matrix factorization model given an RDD of ratings given by users to some products,
-        in the form of (userID, productID, rating) pairs. We approximate the ratings matrix as the
-        product of two lower-rank matrices of a given rank (number of features). To solve for these
-        features, we run a given number of iterations of ALS. This is done using a level of
-        parallelism given by `blocks`.
+        Train a matrix factorization model given an RDD of ratings given by
+        users to some products, in the form of (userID, productID, rating)
+        pairs. We approximate the ratings matrix as the product of two
+        lower-rank matrices of a given rank (number of features). To solve
+        for these features, we run a given number of iterations of ALS. This
```

Comment:
I wonder if the last sentence `This is done using ...` is really necessary (it's better explained in the param doc string below)?